.github/workflows/php.yml

```
@@ -0,0 +1,67 @@
+name: PHP
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-versions: ['7.1', '7.2']
```

Prestonbot runs on php 7.3 ;)
```suggestion
        php-versions: ['7.1', '7.2', '7.3']
```